### DISCUSSION TITLE
Windows or Linux?

### DISCUSSION THREAD

**Discussion by HUSTNO1WXY** [zju3dv/LoFTR#57] [Q&A] [answered]
Windows or Linux?

**Comment by JiamingSuen** [comment, marked as answer]
Linux

**Reply by HUSTNO1WXY** [reply]
Can I run these in Windows?

**Reply by JiamingSuen** [reply]
I'm not sure, since I'm not familiar with running PyTorch-based code in Windows.

**Reply by HUSTNO1WXY** [reply]
I notice that there is "configs/loftr/loftr_ds.py"  in test module of readme. But I can't find this file or directory.

**Comment by deepakkumar-platform** [comment]
> Windows or Linux?
You can run on both OS. Personally, I prefer Linux
